src/ChunkSplitters.jl

```diff
@@ -9,30 +9,45 @@ export chunks, getchunk
 
 This function returns an iterable object that will split the *indices* of `array` into
 to `nchunks` chunks. `type` can be `:batch` or `:scatter`. It can be used to directly iterate
-over the chunks of a collection in a multi-threaded manner.
+over the chunks of a collection in a multi-threaded manner. 
 
-## Example
+The use of `enumerate` provides an indexing of the chunks that can be used to 
+index shared work arrays, buffers, or output collections, without the need of 
+the thread id of the current thread.
+
```

Comment:
Not sure it is necessary to mention `enumerate` here specifically, because it can be combined with every iterator. But why not.

```suggestion
If you need a running chunk index, e.g. to index into a buffer that is shared between chunks, you can combine `chunks` with `enumerate`.
```

Reply:
Actually, what is worth mentioning here, because it is special, is that `enumerate(chunks)` works in conjuction with `@threads` while most `enumerate` applications won't.

Reply:
Yes, exactly.

Reply:
Added comment in https://github.com/m3g/ChunkSplitters.jl/pull/17/commits/66d34b2eb712cfdf6b1e6f2f113afe0c18426ad7